core/felt/felt.go

```diff
@@ -114,6 +115,16 @@ func (z *Felt) String() string {
 	return z.val.String()
 }
 
+// ShortString prints the felt to a string in a shortened format
+func (z *Felt) ShortString() string {
+	hex := z.val.Text(16)
+
+	if len(hex) <= 8 {
+		return fmt.Sprintf("0x%s", hex)
+	}
+	return fmt.Sprintf("0x%s..%s", hex[:4], hex[len(hex)-4:])
```

Comment:
I think we should make it an ellipsis instead `...`

Reply:
done